tests/utils/stl/test/config.py

```diff
@@ -296,22 +259,10 @@ def configure_path_env_var(self):
         self.stl_path_env_var = stl_path_env_var
 
     def configure_test_env(self):
-        stl_test_env = self.get_lit_conf('stl_test_env', None)
-
-        if stl_test_env is None:
-            stl_test_env = self.config.environment
-
-        if self.stl_lib_env_var is None:
-            self.configure_inc_env_var()
-
-        if self.stl_lib_env_var is None:
-            self.configure_lib_env_var()
-
         if self.stl_path_env_var is None:
             self.configure_path_env_var()
 
-        stl_test_env['INCLUDE'] = self.stl_inc_env_var
-        stl_test_env['LIB'] = self.stl_lib_env_var
+        stl_test_env = {}

```

Comment:
How is the freshly-built STL getting picked up by the test infrastructure now? That is, if I replace `<vector>` with `#error WOOF`, will it still cause the tests to fail?

Reply:
I hard-code `/I`s and `/LIBPATH`s.

Reply:
Oh, that was already being done elsewhere, got it.

Reply:
And `PATH` is still getting set to pick up the dlls.